### ISSUE TITLE
Significant background hum

### ISSUE THREAD

**Issue by sawtoothwave** [Normalised/Prok-Drums-Hardware#3]
Loving the sound of the Prok modules so far (built all 4 of them, and then converted the clap to a Klonk), but there's much more background hum/noise than I had expected. I don't mean the low-fi crunch of the 12 bit audio, I mean more like, 60-cycle hum, etc.

I don't think it's from my build since the modules are working perfectly otherwise. It's a little worse in the Snare and Hihat modules, I think, than in the BD and Klonk. It's possible my power supply (Doepfer A-100 DIY kit, so not super high end) isn't helping. Just thought I'd mention to see if it's a known issue, and if there's a chance it's software and could be fixed in a future firmware upgrade.

Thanks!

**Comment by sawtoothwave** [comment]
Just to add a little more context - it's not just hissy noise, which I think I'd be more ok with, it's a very obvious pitched tone. I recorded the background noise from my system and then moved the faders up on the VCA carrying each of the Prok drums one at a time - BD, SD, HH, Klonk (running on a Clap module) as a reference. It starts with 5 seconds of background noise and then moves on to the above in sequence. Thanks!
 
[Prok noise BD SD HH KL.mp3.zip](https://github.com/Normalised/Prok-Drums-Hardware/files/6006447/Prok.noise.BD.SD.HH.KL.mp3.zip)

**Comment by nebularnoise** [comment]
Yeah, I've opened a ticket myself. I'm fairly certain this has to do with the LEDs

**Comment by sawtoothwave** [comment]
Are you hearing the same pitched whine that I am, differing from module to module depending on the firmware? I've been concerned that maybe I just messed up the build, but it doesn't seem like that.

**Comment by nebularnoise** [comment]
yep. I've built 5 proks. They consistently do this, and indeed, the pitch changes with the firmware.

**Comment by sawtoothwave** [comment]
Any thoughts from the Prok team on whether this is something that could/might/will be improved in a future firmware? I really love the sound, usability, and flexibility of these modules—pretty much the only thing I could think of to improve would be adding velocity sensitivity—but the pitched background hum is suuuuuper audible (especially if you compress the drums) in anything but the very busiest mixes.

I'd be very encouraged to hear that this is something that could be addressed in a future firmware, however long it may take for that to happen.

Thank you!

**Comment by milesjohnson777** [comment]
im also having this issue and it even happens when i move my SN out of my custom rig to my tiptop mantis case. i got my module as a diy build so i dont mind performing a hardware mod to fix this if it just needs a capacitor to filter it or something
